crane.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: crane
+  version: 0.12.1
+  epoch: 0
+  description:  Tool for interacting with remote images and registries.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    repositories:
+      - https://packages.wolfi.dev/os
```

Comment:
i think you need to remove repo & keyring. they are appended in ci script

Reply:
So many packages are already using the keyring; this is why I added it too 👇

> https://github.com/search?q=repo%3Awolfi-dev%2Fos+wolfi-signing.rsa.pub&type=code

Reply:
Yeah it was a mistake that those entires were added.

Reply:
do you want me to remove the keyring ?